maintenance/rebuildConceptCache.php

```diff
@@ -100,6 +99,7 @@ protected function addDefaultParams() {
 		$this->addOption( 's', '<startid> Process only concepts with page id of at least <startid>', false, true );
 		$this->addOption( 'e', '<endid> Process only concepts with page id of at most <endid>', false, true );
 
+		$this->addOption( 'runtime', 'Will display the runtime environment of the script', false );
```

Comment:
> runtime environment

This makes me think it's about the PHP version or something like that. Improved in #830